api/xds_protocol.rst

```diff
@@ -449,6 +449,12 @@ no mechanism for providing incremental updates of repeated fields within a named
 Most notably, there is currently no mechanism for incrementally updating individual
 endpoints within an EDS response.
 
+Duplicate Resource Names
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is an error for a server to send a single response that contains the same resource name
+twice.  Clients must NACK responses that contain multiple instances of the same resource name.
```

Comment:
I wonder if we should underspecify this, i.e. say that clients may NACK or chose their own behavior here, e.g. accept first or last resource of the same name. This seems to impose less burden on client implementations for a path that servers should not be following in the first place.

Reply:
Changed "must" to "should".  Does that work?

Reply:
Seems OK, but is there a reason for "should" vs. "may" from your perspective?